apollo-router/src/plugins/response_cache/postgres.rs

```diff
@@ -94,6 +95,9 @@ pub(crate) struct PostgresCacheConfig {
     #[schemars(with = "String")]
     /// Specifies the interval between cache cleanup operations (e.g., "2 hours", "30min"). Default: 1 hour
     pub(crate) cleanup_interval: Duration,
+
+    /// Postgres TLS client configuration
+    pub(crate) tls: Option<TlsConfig>,
```

Comment:
Does `tls: null` mean something different than `tls: {certificate_authorities: null, key: null}` ? If not, let’s remove the outer `Option` and rely on `derive(Default)` and `serde(default)`